kibana.md

```diff
@@ -1001,3 +1001,17 @@ readonlyrest:
 ```
 
 Now try to login as user1, and ReadonlyREST Enterprise should initialise the index ".kibana\_user1" with all the index patterns and dashboards contained in the template tenancy.
+
+## Impersonation
```

Comment:
Shouldn't we add somehow keywords like test settings, and auth mock service into this description?  When the user opens the UI and will see these kinds of words,  and then, will try to search for them, he won't be able to find this description and go into the GitHub page